Handwriting/HandwritingIntentCSSValue.md

```diff
@@ -0,0 +1,176 @@
+
+## Goals
+
+Give authors granular per-document and per-element control over which content should (dis)allow handwriting input.
+
+## Non-Goals
+
+* Allowing websites to determine whether the user agent supports handwriting as an input method, or whether the user has handwriting input enabled.
+* Allowing websites to enable handwriting when it would not otherwise be enabled.
+* Describe how handwriting capabilities should be implemented by browsers or platforms.
+
+## Use Cases
+
+Some scenarios where a website or application may want to disable handwriting input:
+
+* Document editor that wants to temporarily disable handwriting input while certain tools are selected, to support using a stylus to seamlessly draw, place, or size non-text content overtop an editable text region.
+  * Outlook drawing tools with "text pen"<br>![Outlook drawing tools with "text pen" selected](./drawing_tools.png "Outlook drawing tools with \"text pen\" selected")
+* Application with custom form controls that accept sensitive input, have a strict format, or include special characters.
+* Application with custom text inputs or editing experiences that override default browser behaviors by observing and handling input events and editing experiences, doesn't support input method editor (IME) or `composition{start|end|update}` events, or if for any reason the experience designed by website authors doesn't behave as they intend when handwriting input is available.
+  
+
+![example of handwriting input inserting the text "Hello World"](./handwriting.gif "example of handwriting input inserting the text \"Hello World\"")
+
+## Proposed Solution
+
+Introduce a new value, `handwriting`, to the CSS property `touch-action` which allows authors to specify whether an element should allow handwriting input. 
+
+The `touch-action` CSS property is used by authors to define for whether user agents should execute their [direct manipulation](https://w3c.github.io/pointerevents/#dfn-direct-manipulation) behavior for touch and pen gestures. When the spec was written this only included panning and zooming. The `manipulation` value for `touch-action` will be expanded to also indicate that the user agent may consider handwriting interactions on the element.
+
+When the `touch-action` CSS property is specified for an Element, only the mentioned behaviors will be enabled on the Element and all the possible `touch-action` values that are not explicitly mentioned are then disabled for the Element.
+
+Authors are used to the [recommended practice of adding touch-action: none](https://w3c.github.io/pointerevents/#example_10) to elements over which they wish to handle all events themselves.
+
+
+### Syntax
+
+```html
+<style>
+.handwritable {
+  touch-action: handwriting;
+}
+.not-handwritable {
+  touch-action: pan-x;
+}
+</style>
+
+<textarea class="handwritable"></textarea>                                         <!-- the computed handwriting value is true -->
```

Comment:
Can you please have the same alignment for all the comments for better readability?